File: example/src/main/java/com/genius/groupie/example/item/ColumnItem.java
package com.genius.groupie.example.item;

import android.support.annotation.ColorInt;

import com.genius.groupie.example.MainActivity;

public class ColumnItem extends CardItem {

    public ColumnItem(@ColorInt int colorRes, int index) {
        super(colorRes, String.valueOf(index));
        getExtras().put(MainActivity.INSET_TYPE_KEY, MainActivity.INSET);
    }

    @Override public int getSpanSize(int spanCount, int position) {
        return spanCount / 2;
    }
}

File: example-viewbinding/src/main/java/com/xwray/groupie/example/viewbinding/item/ColumnItem.kt
package com.xwray.groupie.example.viewbinding.item

import androidx.annotation.ColorInt
import com.xwray.groupie.example.viewbinding.INSET
import com.xwray.groupie.example.viewbinding.INSET_TYPE_KEY

class ColumnItem(
    @ColorInt colorInt:
    Int, index: Int
) : CardItem(colorInt, index.toString()) {

    init {
        extras[INSET_TYPE_KEY] = INSET
    }

    override fun getSpanSize(spanCount: Int, position: Int): Int = spanCount / 2
}
